src/core/Microsoft.PowerToys.Settings.UI.Lib/SettingsUtils.cs

```diff
@@ -46,29 +47,50 @@ public static bool SettingsExists(string powertoy = DefaultModuleName, string fi
             return File.Exists(GetSettingsPath(powertoy, fileName));
         }
 
+        private static readonly JsonSerializerOptions JsonSerializerOptions = new JsonSerializerOptions() { WriteIndented = true };
+
         /// <summary>
         /// Get a Deserialized object of the json settings string.
         /// </summary>
         /// <returns>Deserialized json settings object.</returns>
         public static T GetSettings<T>(string powertoy = DefaultModuleName, string fileName = DefaultFileName)
         {
-            var jsonSettingsString = File.ReadAllText(GetSettingsPath(powertoy, fileName));
+            var jsonSettingsString = File.ReadAllBytes(GetSettingsPath(powertoy, fileName));
```

Comment:
rename this variable, it is not string but byte[]

Reply:
This is indeed confusing, will change this.